SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -949,112 +915,79 @@ int32 USpatialNetDriver::ServerReplicateActors(float DeltaSeconds)
 
 	FMemMark Mark(FMemStack::Get());
 
-	for (int32 i = 0; i < ClientConnections.Num(); i++)
+	// Only process the fake spatial connection. It will be responsible for replicating all actors, regardless of whether they're owned by a client.
+	USpatialNetConnection* SpatialConnection = Cast<USpatialNetConnection>(ClientConnections[0]);
+	check(SpatialConnection && SpatialConnection->bReliableSpatialConnection);
+
+	// Make a list of viewers this connection should consider (this connection and children of this connection)
+	TArray<FNetViewer>& ConnectionViewers = WorldSettings->ReplicationViewers;
+
+	ConnectionViewers.Reset();
+
+	// The fake spatial connection will borrow the player controllers from other connections.
+	for (int j = 1; j < ClientConnections.Num(); j++)
 	{
-		USpatialNetConnection* SpatialConnection = Cast<USpatialNetConnection>(ClientConnections[i]);
-		check(SpatialConnection);
+		USpatialNetConnection* ClientConnection = Cast<USpatialNetConnection>(ClientConnections[j]);
+		check(ClientConnection);
 
-		if (SpatialConnection->bReliableSpatialConnection || SpatialConnection->ViewTarget)
+		if (ClientConnection->ViewTarget != nullptr)
 		{
-			// Make a list of viewers this connection should consider (this connection and children of this connection)
-			TArray<FNetViewer>& ConnectionViewers = WorldSettings->ReplicationViewers;
+			new(ConnectionViewers)FNetViewer(ClientConnection, DeltaSeconds);
 
-			if (SpatialConnection->ViewTarget)
+			for (UChildConnection* ChildConnection : ClientConnection->Children)
```

Comment:
Realise this isn't your code, but we don't use child connections - shouldn't we assert here instead?